src/shared/DashboardLaunchButton/index.jsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
+import { logError } from '@edx/frontend-platform/logging';
+import { Button } from 'react-paragon-topaz';
+
+import AlertMessage from 'shared/AlertMessage';
+import { xtremeLabsUrl, defaultErrorMessage } from 'constants';
+import { eventManager } from '../../helpers';
```

Comment:
Can we use the alias path here?

```suggestion
import { eventManager } from 'helpers';
```